### PR TITLE
Add documentation for server upgrade

### PR DESCRIPTION
**Pull request by cawallin**
Testing: make docs

### CODE REVIEW

**Comment by cawallin** [comment]
@rschlussel @mattsfuller

File: docs/presto-admin-commands.rst [hunk 1]
```diff
@@ -131,6 +131,27 @@ Example
...
 
 
+**************
+server upgrade
+**************
+::
+
+    presto-admin server upgrade local_package_path [local_config_dir]
+
+This command upgrades the Presto RPM on all of the nodes in the cluster to the RPM,
```
Comment (rschlussel) [inline docs/presto-admin-commands.rst]: this sentence is confusing.  maybe say "to the rpm specified by local_package_path.

Comment (rschlussel) [inline docs/presto-admin-commands.rst]: Also mention that it will downgrade the version too if the local_package_path rpm is an earlier version.

**Comment by cawallin** [comment]
updated

**Comment by rschlussel** [comment]
lgtm

**Pull request merged**